addon/components/detail.js

```diff
@@ -101,6 +101,7 @@ export default Component.extend(PropTypeMixin, {
     ]),
     hook: PropTypes.string,
     onChange: PropTypes.func,
+    tabName: PropTypes.string,
```

Comment:
This property name isn't clear as to what it is doing. Can we rename this to be `selectedTabLabel`.